src/NativeRNCImageEditor.ts

```diff
@@ -49,6 +49,11 @@ export interface Spec extends TurboModule {
        * (Optional) The format of the resulting image. Default auto-detection based on given image
        */
       format?: string;
+
+      /**
+       * (Optional) Indicates if Base64 formatted picture data should also be included in the result.
+       */
+      base64?: boolean;
```

Comment:
If the value is a boolean flag, could you change its name to sth like : `includeBase64` or `isBase64` or `shouldReturnBase64` ?

Reply:
no problem,  `includeBase64` was applied instead